.github/actions/azure/action.yml

```diff
@@ -0,0 +1,45 @@
+# This GitHub Action is designed to execute Azure CLI for infrastructure maintenance
+
+name: 'Deploy Azure WebApp'
+description: 'This custom GitHub Actions module will execute AZ CLI for infrastructure maintenance'
+
+# Define the inputs required for this action.
+inputs:
+  credentials:
+    description: 'Azure subscription and resource group service principal'
+    required: true
+  subscription:
+    description: 'Azure subscription ID'
+    required: true
+  group:
+    description: 'Azure resource group'
+    required: true
+  acr:
+    description: 'Azure subscription and resource group service principal'
+    required: true
+  days:
+    description: 'Retention days for ACR artifacts purge i.e images older than x days will be removed.'
+    required: true
```

Comment:
if there's default, do we need required: true?

Reply:
Good spot, have updated.